streaming_kafka_consumer/codecs.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
```

Comment:
meh, it is unfortunate we need to duplicate this.
Well I guess there is no alternative, and this is small enough.